spec/pipelines/pacbio/message_spec.rb

```diff
@@ -112,7 +112,7 @@
           end
 
           it 'can have a tag set name' do
-            expect(message_sample[:tag_set_name]).to eq(request_library.tag.set_name)
+            expect(message_sample[:tag_set_name]).to eq(request_library.tag.tag_set.name)
           end
```

Comment:
Doesn't really matter but can use the `tag_set_name` on request libraries again 😊